svc-bip-api/src/main/java/gov/va/vro/bip/service/RMQController.java

```diff
@@ -0,0 +1,38 @@
+package gov.va.vro.bip.service;
+
+import gov.va.vro.bip.model.*;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class RMQController {
+  @Autowired BipApiService service;
+
+  @RabbitListener(queues = "getClaimDetailsQueue", errorHandler = "xampleErrorHandler")
```

Comment:
I think all the error handlers on these `@RabbitListener` methods need to be updated to the corresponding error handler bean. For example this one should be updated to `@RabbitListener(queues = "getClaimDetailsQueue", errorHandler = "errorHandlerForGetClaimDetails"`

Reply:
It was initially `errorHandlerForGetClaimDetails` but I changed it to `xampleErrorHandler` because of this comment https://shorturl.at/fgD59.   Please talk to Tom about which way you all want it to be.

Reply:
That's fine with me, but that xampleErrorHandler is pointing directly to the bean by that name. If that works for all of these listeners then there isnt an issue and there is no point in declaring different error handlers for each queue.

Reply:
depends on what is meant by "works,"  I guess.

Reply:
I initially decide to have multiple handlers because methods in the service have different return values.  And it appeared that error handler has to return the same type as the `@RabbitListener` that caused the exception.  I was getting serialization error otherwise.